packages/design-system/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,168 @@
+import cx from 'classnames';
+import SearchIcon from 'react-material-icon-svg/dist/MagnifyIcon';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+import { debounce } from '@livechat/data-utils';
+
+import styles from './style.scss';
+import { Loader } from '../Loader';
+import { KeyCodes } from '../../constants/keyCodes';
+
+const baseClass = 'search-bar';
+const noop = () => {};
+
+class SearchBar extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.inputRef = React.createRef();
+    this.onChangeDebounced = debounce(this.props.debounceInMs, value => {
+      this.props.onChange(value);
+    });
+
+    this.state = {
+      searchTerm: '',
+      isInCompactMode: this.props.compact
+    };
+  }
+
+  handleChange = e => {
+    const { value } = e.target;
+    const { debounceInMs, onChange } = this.props;
+
+    this.setState({
+      searchTerm: value
+    });
+
+    if (debounceInMs) {
+      this.onChangeDebounced(value);
+    } else {
+      onChange(value);
+    }
+  };
+
+  handleKeyPress = key => {
+    const { onSubmit } = this.props;
+
+    if (onSubmit) {
+      if (key.keyCode === KeyCodes.enter) {
+        onSubmit(this.state.searchTerm);
+      }
+    }
+  };
+
+  handleClear = () => {
+    this.setState({
+      searchTerm: ''
+    });
+
+    this.inputRef.current.focus();
+  };
+
+  handleKeyDown = key => {
+    if (key.keyCode === KeyCodes.enter) {
+      this.handleClear();
+    }
+  };
+
+  toggleCompactMode = () => {
+    this.setState(state => ({
+      isInCompactMode: !state.isInCompactMode
+    }));
+  };
+
+  render() {
+    const {
+      className,
+      placeholder,
+      value,
+      loading,
+      compact,
+      error,
+      onChange,
+      onSubmit,
+      ...restProps
+    } = this.props;
+
+    const { searchTerm, isInCompactMode } = this.state;
+
+    const shouldDisplayCloseButton = searchTerm && !loading && !isInCompactMode;
+
+    return (
+      <span className={className}>
+        <div className={styles[`${baseClass}__container`]}>
+          <SearchIcon
+            fill="#424d57"
+            width="18px"
+            height="18px"
+            className={cx(`lc-${baseClass}__icon--search`, {
+              [`lc-${baseClass}__icon--search-compact`]: compact
+            })}
+            onClick={compact ? this.toggleCompactMode : null}
+          />
+          {loading && (
+            <Loader
+              size="small"
+              className={styles[`${baseClass}__icon--loader`]}
+            />
+          )}
+          <input
+            type="input"
+            placeholder={placeholder}
+            ref={this.inputRef}
+            value={value || this.state.searchTerm}
+            onInput={this.handleChange}
+            onKeyDown={this.handleKeyPress}
+            className={cx(`lc-${baseClass}__input`, {
+              [`lc-${baseClass}__input-compact`]: isInCompactMode
+            })}
+            {...restProps}
```

Comment:
<img width="698" alt="Monosnap 2020-08-11 11-23-27" src="https://user-images.githubusercontent.com/24607370/89880934-1f656380-dbc5-11ea-9369-64bb85bab54b.png">
you need to omit this property